src/meter/Meter.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import FlexView from '../flex/FlexView';
+import find from 'lodash/collection/find';
+import every from 'lodash/collection/every';
+import uniq from 'lodash/array/uniq';
+
+const Range = t.refinement(t.struct({
+  startValue: t.Number,
+  endValue: t.Number,
+  fillingColor: t.maybe(t.String),
+  labelColor: t.maybe(t.String)
+}), r => r.startValue < r.endValue, 'Range');
+
+const Ranges = t.refinement(t.list(Range), (rangeList) => {
+
+  const stepsWithout = (steps, stepToRemove) => {
+    const stepsNoDuplicate = uniq(steps);
+    return stepsNoDuplicate.filter( step => (
+      !(step.startValue === stepToRemove.startValue && step.endValue === stepToRemove.endValue)
+    ));
+  };
+
+  const isOverlapped = (step1, step2) => (
+    Math.max(step1.startValue, step2.startValue) < Math.min(step1.endValue, step2.endValue)
+  );
+
+  const noOverlappingRanges = (steps) => {
+    return every((steps), step1 => {
+      return every(stepsWithout(steps, step1), step2 => {
+        return !isOverlapped(step1, step2);
+      });
+    });
+  };
+
+  return noOverlappingRanges(rangeList);
+}, 'Ranges');
+
+const Props = t.refinement(t.struct({
+  /**
+   * This is the value provided as input.
+   */
+  current: t.Number,
+  /**
+   * Minimum value.
+   */
+  min: t.maybe(t.Number),
+  /**
+   * Maximum value.
+   */
+  max: t.maybe(t.Number),
+  /**
+   * Function as input in which you can define a custom label format.
+   */
+  labelFormatter: t.maybe(t.Function),
+  /**
+   * Array of Object in which you can define startValue, endValue, labelColor, fillingColor.
+   */
+  steps: t.maybe(Ranges),
+  defaultLabelColor: t.maybe(t.String),
+  defaultFillingColor: t.maybe(t.String),
+  id: t.maybe(t.String),
+  className: t.maybe(t.String),
+  style: t.maybe(t.Object)
+}), ({ min, max, steps }) => {
+  const minStartValue = Math.min(...steps.map(r => r.startValue));
+  const maxStartValue = Math.max(...steps.map(r => r.endValue));
+
+  return min === minStartValue && max === maxStartValue;
+}, 'Props');
+
+const computeResult = (current, min, max) => {
+  return Math.abs((current - min) * 100 / (max - min));
+};
+
+const labelFormatter = (current, min, max) => {
+  return `${computeResult(current, min, max)}%`;
+};
+/**
+ * ### Renders a Progress Bar
+ */
+@skinnable()
+@props(Props)
+export default class Meter extends React.Component{
+
+  static defaultProps = {
+    min: 0,
+    max: 100,
+    defaultFillingColor: '#ccc',
```

Comment:
I know, what I'm saying is it makes no different having the HTML default `transparent` instead of yours arbitrary `#CCC` In both case it's a defect as it's very unlikely the user wants `#CCC`. That's why I suggest you replace this default with a warning